Makefile

```diff
@@ -120,7 +120,7 @@ prove : $(tests)
 examples : $(examples)
 
 .PHONY: clean print test examples astyle
-clean :
+cln clean :
```

Comment:
Do we need this alias? If it's important, it should also go in `.PHONY` but I'm inclined to remove it.